src/components/SignUpForm/Form.tsx

```diff
@@ -38,6 +39,9 @@ const Form = () => {
         <label className="form__input-label">Confirm Password</label>
         <input type="text" className="form__input" />
       </div>
+      <div className="form__recaptcha">
+        <ReCAPTCHA sitekey={process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY!} />
```

Comment:
Quick question why do we need the `!`?

Reply:
To solve the problem about `undefined` type. 273

We can now use the [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator). It tells TypeScript that even though something looks like it could be null, it can trust us that it's not.

Reply:
Alternatively, we can choose to render this `<ReCAPTHCA />` component only if the `process.env.NEX...` is not undefined. The solution can be found in [react conditional rendering](https://reactjs.org/docs/conditional-rendering.html)

Reply:
As I said earlier, we definitely declare the environment variable (sitekey) but in Github we don't have one, so you have to use your local env instead. Anyway, in Docker, I already declared it. Surely, the site key never be undefined.